treeherder/webapp/api/runnable_jobs.py

```diff
@@ -57,6 +78,31 @@ def list(self, request, project):
                 'state': 'runnable',
                 'result': 'runnable'})
 
+        for node in tc_graph['tasks']:
```

Comment:
you might set a var of ``task = node['task']`` so that you don't have to reference node['task'] everywhere..  not critical, though...